groclient/client_test.py

```diff
@@ -479,6 +479,14 @@ def test_convert_unit(self):
             self.client.convert_unit({"value": 1, "unit_id": 37}, 36),
             {"value": -17.5, "unit_id": 36},
         )
+        self.assertEqual(
+            self.client.convert_unit({"value": 20, "unit_id": 10, "metadata": {"conf_interval": 2}}, 14),
+            {"value": 0.02, "metadata": {"conf_interval": 0.002}, "unit_id": 14},
+        )
+        self.assertEqual(
+            self.client.convert_unit({"value": 20, "unit_id": 10, "metadata": {"conf_interval": None}}, 14),
```

Comment:
I think we barely have the case with `"metadata": {"conf_interval": None}`. If no confidence interval is provided, it would rather leave that attribute out. On the other hand, you might want to add a test case with `"metadata": {}`